source/versioned-api/tests/runcommand-helper-no-api-version-declared.json

```diff
@@ -4,6 +4,11 @@
   "runOnRequirements": [
     {
       "minServerVersion": "4.7"
+    },
```

Comment:
The requirements are OR'd together.  In this case, we need an AND:

     {
       "minServerVersion": "4.7",
       "serverParameters": {
          "requireApiVersion": false
       }
     }

Reply:
Whoopsie - there are things I shouldn't do past 8 pm. Thanks for noticing, I've updated this.

Reply:
Do you have an environment where you can execute these tests?  I didn't catch this by inspection, I had to run them, and even then it took me a minute to spot it.